pageObject/checkOutStep1.js

```diff
@@ -0,0 +1,32 @@
+const Page = require("./page");
+
+class CheckoutStep1 extends Page {
```

Comment:
Better to avoid numbers in names. Could it be CheckoutPage?

Reply:
thank you! renamed to CheckoutPage